### PR TITLE
Added preview and route-bundle

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR includes the new sulu-bundles preview and route. The main PR can be found here https://github.com/sulu/sulu/pull/2299

### CODE REVIEW

**Comment by chirimoya** [comment]
@wachterjohannes ... tests are failing

**Comment by wachterjohannes** [comment]
@chirimoya the smart-content seems not to trigger the seva-button all the over things seems to work.

**Comment by wachterjohannes** [comment]
@chirimoya this PR also includes the change for default disabled websocket. the problem was that `$rootNode->canBeEnabled()` automatically enables it if a child config is set (like the server here). so we have to change the user config.

File: app/config/admin/config.yml [hunk 1]
```diff
@@ -7,3 +7,6 @@ imports:
 # Assetic Configuration
 assetic:
     bundles:        [ ClientWebsiteBundle ]
+
```
Comment (chirimoya) [inline app/config/admin/config.yml]: we shouldn't add any sulu specific config to this file. this there another solution?

**Pull request merged**